Meadowlark-js/packages/meadowlark-core/src/handler/FrontendFacade.ts

```diff
@@ -31,13 +33,15 @@ function postStack(): MiddlewareStack {
       R.andThen(parsePath),
       R.andThen(parseBody),
       R.andThen(resourceValidation),
+      R.andThen(metaeEdModelFinding),
```

Comment:
Also split out model finding. It was needed in three different places, so made a separate middleware. However, adding the found MetaEd model to the frontendRequest is what initially caused logging issues.